File: examples/rb.cpp
sim.set_num_qubits(2);
std::vector<int> creg(2);
// Unsupported operation: reset
// Unsupported operation: reset
sim.hadamard(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: barrier
sim.gate_z(1, std::vector<int>{}, std::vector<int>{0});
// Unsupported operation: barrier
sim.gate_s(0, std::vector<int>{}, std::vector<int>{});
sim.gate_z(1, std::vector<int>{}, std::vector<int>{0});
// Unsupported operation: barrier
sim.gate_s(0, std::vector<int>{}, std::vector<int>{});
sim.gate_z(0, std::vector<int>{}, std::vector<int>{});
sim.hadamard(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: barrier
creg[0] = sim.measure(0);
creg[1] = sim.measure(1);

File: examples/rb.py
from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister

q = QuantumRegister(2, "q")
c = ClassicalRegister(2, "c")
qc = QuantumCircuit(q, c)

qc.reset(q)
qc.h(q[0])
qc.barrier(q)
qc.cz(q[0], q[1])
qc.barrier(q)
qc.s(q[0])
qc.cz(q[0], q[1])
qc.barrier(q)
qc.s(q[0])
qc.z(q[0])
qc.h(q[0])
qc.barrier(q)

qc.measure(q, c)
